tests/util.py

```diff
@@ -3,6 +3,27 @@
 
 from uuid import uuid4
 from hazelcast.config import SSLProtocol
+from hazelcast.six import PY3
+
+
+def get_current_timestamp():
```

Comment:
Instead of defining this as a function, and checking whether or not `time` has `monotonic` attribute each time this is called, we can define this function like
```python
if hasattr(time, "monotonic"):
    get_current_timestamp = time.monotonic
else:
    get_current_timestamp = time.time
```

Note that, we don't need to check for PY3 anymore, we will simply use it if we have it

Reply:
Ok check it again pls